tests/nnstreamer_decoder_image_segment/runTest.sh

```diff
@@ -78,6 +78,26 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! videoc
 
 callCompareTest test_golden.0 test_output.0 1 "test with videotestsrc" 0
 
+# THIS WON'T FAIL, BUT NOT MUCH MEANINGFUL (TARGET MODEL IS DIFFERENT ONE)
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
+videotestsrc num_buffers=1 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=257,height=257 ! tee name=t \
+    t. ! queue ! mix. \
+    t. ! queue ! tensor_converter ! tensor_transform mode=arithmetic option=typecast:float32,div:255.0 ! \
+        tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} latency=1 throughput=1 ! \
```

Comment:
Q: what means 'latency=1 throughput=1'?

Reply:
They're recently added properties for profiling tensor_filter. App may get invoke latency information from this.
Details: #2417, #2422

Reply:
Maybe we need to rename them later: e.g., ```profile_latency=1 profile_throughput=1``` or ```profile=latency,throughput``` / ```profile=no```.